apis/nucleus/src/viz.js

```diff
@@ -149,6 +150,51 @@ export default function viz({ model, halo, initialError, onDestroy = async () =>
       const propertyTree = await conversionConvertTo({ halo, model, cellRef, newType });
       return propertyTree;
     },
+    /**
+     * Toggles the view of the chart into an accessible table
```

Comment:
Maybe explain that the type you want to toggle to, sn-table by default, needs to be registered on the embed instance to be able to do this?

Reply:
Added some more info and also a check if the type is actually registered